Heart/Character/Domain/Entities/LevelEntity.php

```diff
@@ -69,6 +69,11 @@ public function generateExperience(string $message, bool $isSupporter = false):
         $averageMessageLength = 25;
         $memberStatusMultiplier = $isSupporter ? 0.25 : 0.4;
         $messagePound = ($messageLength / $averageMessageLength);
+
+        if ($this->level === 0) {
+            $this->level += 1;
```

Comment:
@Canhassi12 Meus 50 cents aqui, na real como você está modificando o valor do level, próximos usos da instancia dessa LevelEntity (na CharacterEntity->getLevel), vai assumir que a pessoa já está no level 1. O ideal seria somente tratar o cenario em uma variável local com o $level.

E `+= 1` não é necessário, pois se `level === 0`, `+= 1` sempre vai ser 1. Logo, `$this->level = 1;`

Reply:
@anibalsolon, Faz sentido. Fiz de outra forma para corrigir de um jeito que não tenha códigos repetidos, como por exemplo repetir os calculos dentro do IF utilizando um level, e um fora utilizando outro. O que acha?